README.md

```diff
@@ -90,6 +91,13 @@ This extension contributes the following settings:
 | NSFW only check Template                    | `!nsfwonly`  | Sets a command as nsfw only           |
 | Has Permissions check Template              | `!hasperms`  | makes a command require a permission  |
 
+### UI
+
+| Name                                        | Prefix       | Description                           |
+|---------------------------------------------|--------------|---------------------------------------|
+| Button                                      | `!button`    | A basic button                        |
+| Custom Button                               | `!cbutton`   | A button with a custom constructor    |
+| Select                                      | `!select`    | A basic select                        |
```

Comment:
```suggestion
| Button                                      |  `!button`   | A basic button                        |
| Custom Button                               |  `!cbutton`  | A button with a custom constructor    |
| Select                                      |  `!select`   | A basic select                        |
```

Although this doesn't matter but I like to keep the prefix centered if for some reason someone reads the actual raw markdown